install/install.sh

```diff
@@ -40,6 +40,9 @@ do
                 if [ "$mysql_user_resp" = "Q" ]; then
                     exit 1
                 fi
+            fi
+			if [  -f "$file" ]; then
```

Comment:
Your alignment is not ok. Please correct.

Reply:
I will create a new PR for the alignment